src/guides/v2.3/ui_comp_guide/components/ui-columns-editing-client.md

```diff
@@ -0,0 +1,71 @@
+---
+group: ui-components-guide
+title: ColumnsEditingClient component
+contributor_name: Atwix
+contributor_link: https://www.atwix.com/
+---
+
+The ColumnsEditingClient [UI component](https://glossary.magento.com/ui-component) is an [extension](https://glossary.magento.com/extension) for the [ColumnsEditor]({{ site.mage2bloburl }}/{{page.guide_version}}/app/code/Magento/Ui/view/base/web/js/grid/editing/editor.js) component, it provides functionality for saving edited data.
```

Comment:
..of the ColumnsEditor component. It provides. (Period at end of sentence. Start with uppercase i.